helm/azuredisk-csi-driver-app/templates/_helpers.tpl

```diff
@@ -11,10 +11,21 @@
 
 {{/* labels for helm resources */}}
 {{- define "azuredisk.labels" -}}
-app.kubernetes.io/instance: "{{ .Release.Name }}"
-app.kubernetes.io/managed-by: "{{ .Release.Service }}"
-app.kubernetes.io/name: "{{ template "azuredisk.name" . }}"
-app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
-helm.sh/chart: {{ include "chart" . | quote }}
-application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+labels:
+  app.kubernetes.io/instance: "{{ .Release.Name }}"
+  app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+  app.kubernetes.io/name: "{{ template "azuredisk.name" . }}"
+  app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+  helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+  application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+{{- end -}}
```

Comment:
This looks wrong. Isn't `labels` already in the manifest so we don't have to repeat it here?